lua/mason-registry/remark-cli/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "remark-cli",
+    desc = [[Command line interface to inspect and change markdown files with remark.]],
+    homepage = "https://github.com/remarkjs/remark/tree/main/packages/remark-cli",
+    languages = { Pkg.Lang.Markdown },
+    categories = { Pkg.Cat.LSP },
```

Comment:
```suggestion
    categories = { Pkg.Cat.Formatter },
```

or am I missing something?

Reply:
No, just copy/paste error on my part.